tools/foldseek_tutorial.md

```diff
@@ -0,0 +1,59 @@
+# Foldseek
+written by: [Izumi Ando](https://github.com/izumiando)
+
+[10 minutes] This is a tutorial of Foldseek, a tool that allows you to search through multiple databases for similar proteins based on structure, not just alignment. It takes in either a protein sequence or a PDB file / accession and returns a list of proteins that are similar in structure by doing an alignment of the 3Di sequence. (3Di sequence : The developers created an alphabet that captures the nearest neighbor interactions of a protein which is used in Foldseek to find proteins similar in structure but have very different protein sequences.) This tool can be used for Virus Discovery as a starting point to find hints on the characterisitcs of the Oblin the Obelisk encodes and to find structurally similar viruses.
+
+**Tutorial Objective**: Pick up hints on how to find a protein with a structure that is similar to the Oblin encoded by Izumi's Obelisk.
+
+## Input / Prerequisites
+- [Tool Weblink](https://search.foldseek.com/)
+- Required Inputs: either a protein structure (PDB/CIF), PDB accession, or protein sequence (FASTA)
+- The webserver provided for this tool should be enough for most users, but if you would like to install the tool on your computer you can refer to their GitHub for installation & set up. [Fold Seek GitHub](https://github.com/steineggerlab/foldseek)
+- Sample Input: Izumi's Obelisk's protein sequence generated with the Biostring R package  [Download Here](https://drive.google.com/file/d/1oL9xgQQk4j44UJfJKJd1MtCDoJumQos5/view?usp=sharing)
+- [R script used to generate the protein sequence above](https://drive.google.com/file/d/1m8uXUS3BAtTPnSg0PWIGKHg0fJr-Wiau/view?usp=sharing)
+
+## Output
+
+Foldseek will return a list of proteins similar in structure to the query. For each result, the user can explore each result by looking at the protein sequence alignemnt, viewing the superposed 3D structure, and checking the TM score all within the online tool. You can also download the **PDB structure** or its **PNG** to use outside this tool in software such as Chimera or pymol. 
+
+### 1. Download protein sequence FASTA file & upload to Foldseek
+
+- First, download the protein sequence FASTA file from the link provided above. This protein sequence was generated from the whole nucleotide sequence of Obelisk_001739_000001_000001 using the R package Biostrings. The script used for this also provided above for reference and reproducibility. 
+- Next, drag the FASTA file into the input box and click "Predict" for "Structure with ESMFold". This will call the ESMFold protein structure predictor made by Meta to predict the structure of the Obelisk. This step is necessary because there are no structures currently available for this Obelisk. If there was, (for example from a previous AlphaFold run,) the structure from that could be used as a search query instead in Step 2 of this tutorial.
+
+![](./img/Foldseek/predict_structure.png)
```

Comment:
For each image, the `./` is not needed you should link to the images as `img/Foldseek/predict_structure.png`. Also the "Figure Title" or Figure Text here is not provided so make sure there is text incase the image does not load, that a description is still available.